vue-app/src/components/RoundInformation.vue

```diff
@@ -369,6 +391,10 @@ export default class RoundInformation extends Vue {
     return this.$store.state.currentRound
   }
 
+  get joinTimeLeft(): TimeLeft {
+    return getTimeLeft(this.$store.getters.recipientJoinDeadline)
```

Comment:
Good question. I think what you have makes sense (countdown to the `recipientJoinDeadline`).

The reason we have multiple getters is because it's possible for the join phase & the contribution phase to overlap but there's a time period where it's ONLY the join phase (`isRoundJoinOnlyPhase`).